### PR TITLE
Reformat everything with scalafmt using vertical alignment.

### PR DESCRIPTION
**Pull request by olafurpg**
Reformatted everything with scalafmt using vertical alignment. Why? Because our time is better spent on solving problems than discuss trivia. I admit the output from scalafmt in this diff is not always perfect, but it's good enough for my personal taste. Please comment on the diffs you don't like, maybe I can fix those issues :)

### CODE REVIEW

File: .scalafmt.conf [hunk 1]
```diff
@@ -0,0 +1,7 @@
+style = defaultWithAlign
+maxColumn = 100
+optIn.breakChainOnFirstMethodDot = true
+project.git = true
+project.excludeFilters = [
+  "plugin/src/main/scala/org/scalameta/paradise/backend/Backend.scala"
```
Comment (olafurpg) [inline .scalafmt.conf]: We can exclude files like this.

File: plugin/src/main/scala/org/scalameta/paradise/reflect/LogicalTrees.scala [hunk 2]
```diff
@@ -236,7 +249,9 @@ trait LogicalTrees {
       def unapply(tree: g.Tree): Option[l.Template] = tree match {
         case g.Apply(g.Select(g.New(tpt), nme.CONSTRUCTOR), args) =>
           val lparent = g.Apply(tpt, args).set(new SupercallRole)
-          val lself = g.ValDef(g.Modifiers(), g.nme.WILDCARD, g.TypeTree(), g.EmptyTree).set(SelfRole(g.EmptyTree))
+          val lself =
+            g.ValDef(g.Modifiers(), g.nme.WILDCARD, g.TypeTree(), g.EmptyTree)
```
Comment (olafurpg) [inline plugin/src/main/scala/org/scalameta/paradise/reflect/LogicalTrees.scala]: Is this better @xeno-by ? I put a newline before `g`. Sometimes you need to massage scalafmt a bit to get better output, this is especially common for infix operators.

Comment (xeno-by) [inline plugin/src/main/scala/org/scalameta/paradise/reflect/LogicalTrees.scala]: Yeah it is, I think

File: .scalafmt.conf (hunk 1 above)
Comment (xeno-by) [inline .scalafmt.conf]: Great!

Comment (olafurpg) [inline .scalafmt.conf]: This feature is brand new, worked on it last saturday

**Comment by xeno-by** [comment]
Superseded by https://github.com/scalameta/paradise/pull/66

**Pull request closed without merging**